### PR TITLE
[17.01] Correct label of deprecated data libraries in admin menu

### PR DESCRIPTION
**Pull request by martenson**

### CODE REVIEW

**Comment by dannon** [comment]
I was trying to look up what functionality still needs to be implemented and found this issue from 2015 -- is https://github.com/galaxyproject/galaxy/issues/1177 still up-to-date?

**Comment by martenson** [comment]
@dannon give or take, I am trying to keep the first checklist fairly updated to what was implemented and what is missing and requested. 

edit: Not all of those requests listed are present in the old UI though.

**Comment by dannon** [comment]
Ok, cool, thanks!  (Just trying to fill my context in here a bit)

I'm not sure we can call it deprecated yet, but if that decision has already been made, then this just needs one extra article -- "for *the* majority of use cases".  Or, maybe the simpler "For most use cases"?

File: templates/webapps/galaxy/admin/center.mako [hunk 1]
```diff
@@ -88,7 +88,9 @@ Please visit <a href="https://wiki.galaxyproject.org/Admin" target="_blank">the
                 </li>
             %endif
                 <li>
-                    <strong>Data libraries</strong> - Data libraries enable authorized Galaxy users to share datasets with other groups or users. Only administrators can create data libraries. See <a href="https://wiki.galaxyproject.org/DataLibraries" target="_blank">wiki</a> for details and <a href="https://wiki.galaxyproject.org/Admin/DataLibraries/LibrarySecurity" target="_blank">this page</a> for security description.
+                    <strong>Data libraries</strong> - This view is <strong>deprecated</strong>, for the majority of use cases please use the interface under Shared Data in the top menu.
+                    <br/>
+                    Data libraries enable authorized Galaxy users to share datasets with other groups or users. Only administrators can create data libraries. See <a href="https://wiki.galaxyproject.org/DataLibraries" target="_blank">wiki</a> for details and <a href="https://wiki.galaxyproject.org/Admin/DataLibraries/LibrarySecurity" target="_blank">this page</a> for security description.
```
Comment (nsoranzo) [inline templates/webapps/galaxy/admin/center.mako:93]: Unfortunately [this wiki page](https://wiki.galaxyproject.org/Admin/DataLibraries/LibrarySecurity) seems to not be available in the new wiki yet.

**Pull request closed without merging**